syft/frameworks/torch/hook/hook.py

```diff
@@ -841,7 +861,6 @@ def _which_methods_should_we_auto_overload(self, tensor_type: type):
         """Creates a list of Torch methods to auto overload.
 
         By default, it looks for the intersection between the methods of
-        tensor_type and torch_type minus those in the exception list
         (syft.torch.exclude).
```

Comment:
A line has vanished

Reply:
Which line is no longer there exactly?

Reply:
844: tensor_type and torch_type minus those in the exception list

Reply:
@mari-linhares Thank you for putting it back there!